src/server/app/util/services/util.server.service.js

```diff
@@ -176,7 +176,7 @@ module.exports.getPage = function (queryParams) {
  * Extract given field from request header
  */
 module.exports.getHeaderField = function (header, fieldName) {
-	return (null == header || null == header[fieldName]) ? null : header[fieldName];
+	return (null == header || null == header[fieldName]) ? null : decodeURI(header[fieldName]);
```

Comment:
@ymao1 See the Interface-URL question. You introduced it 4 months ago: https://github.com/Asymmetrik/mean2-starter/commit/d929a042de6b64604cc18f93226b7fa367a1e566